Source/Scene/Scene.js

```diff
@@ -417,22 +427,13 @@ define([
         this.debugShowFramesPerSecond = false;
 
         /**
-         * If <code>true</code>, enables Fast Aproximate Anti-aliasing only if order independent translucency
-         * is supported.
-         *
-         * @type Boolean
-         * @default true
-         */
-        this.fxaaOrderIndependentTranslucency = true;
-
-        /**
          * When <code>true</code>, enables Fast Approximate Anti-aliasing even when order independent translucency
          * is unsupported.
          *
          * @type Boolean
-         * @default false
+         * @default true
          */
-        this.fxaa = false;
+        this.fxaa = true;
```

Comment:
If we aren't copying the depth, then it doesn't need to be, but then there will need to be two paths depending on what the option to copy the globe depth is set to.